db/src/tx.rs

```diff
@@ -254,13 +254,13 @@ impl<'conn, 'a> Tx<'conn, 'a> {
 
                     let v = match v {
                         entmod::AtomOrLookupRefOrVectorOrMapNotation::Atom(v) => {
-                            if attribute.value_type == ValueType::Ref && v.is_text() {
-                                Either::Right(LookupRefOrTempId::TempId(temp_ids.intern(v.as_text().cloned().map(TempId::External).unwrap())))
+                            if attribute.value_type == ValueType::Ref && v.inner.is_text() {
```

Comment:
I don't think we can, since we'd need to provide an `&Value`.  To do so you need to recurse into the children of `SpannedValue` in order to convert them to `&Value`, and I don't think we can produce an owned container with a sufficiently long lifetime to collect the deref-ed children.  I'd be happy to be proven wrong -- and thanks for bringing `Deref` to my attention!